raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+PART IX - WASTE MANAGEMENT
+
+[a] Management of Solid Waste 
+
+Duty of local government to manage and minimize solid waste 
+(114) [1] For the purposes of ensuring minimization of the solid waste in their respective geographical areas of jurisdiction, local government authorities shall prescribe: [a] for different types or kinds of waste or refuse or garbage to be separated at the source; [b] for standards to guide the type, size, shape, colour and other specifications for refuse containers used; and [c] for mechanisms to be put in place to involve the private sector and Non-Governmental Organisations on planning, raising awareness among producers, vendors, transporters, manufacturers and others on the need to have appropriate containers and enhance separation of waste at source. [2] The local government authorities shall, with respect to their respective geographical areas of jurisdiction: [a] cause to be conducted appropriate Environmental Impact Assessment for all new major activities leading to proper management of solid waste; [b] manage solid waste generated in accordance with sustainable plans produced by respective local government authority; and [c] ensure the appropriate sorting of waste is made right at the source and in accordance with standards or specifications prescribed by the local government authority concerned. [3] For the purposes of making decisions on suitability of any specific method of collection, treatment or disposal of solid waste, local government authorities shall undertake studies within their respective geographical areas of jurisdiction with a view to instill knowledge of the rate of local solid waste generation and its composition. 
+
+Disposal of solid waste from markets, business areas and institutions 
+(115) [1] Each local government authority shall undertake periodic studies to determine the type of solid wastes generated from markets, business areas and institutions and determine appropriate methods for sorting, storage or disposal of waste. [2] In the determination of the appropriate storage or disposal for solid waste generated by different types of markets, business centres or areas and institutions within their respective areas, local government authorities shall ensure that the solid waste is classified and appropriately stored depending on whether it is organic waste, plastics, glass or metals. [3] The Minister may prescribe national classifications of solid waste which local government authorities may adopt for their respective geographical areas of jurisdictions. [4] for the purpose of this section, "institution" includes schools, offices, hospitals, police, army barracks, religious buildings, camps, convents and any other organization or entity which the Minister may declare to be an institution for the purposes of solid waste management. 
```

Comment:
Please replace:
* `geographical areas of jurisdictions` with `geographical areas of jurisdiction`; and
* `[4] for` with `[4] For`.